### ISSUE TITLE
Error detected while processing function <SNR>42_InstallCLI

### ISSUE THREAD

**Issue by ivanlp10n2** [wakatime/vim-wakatime#141]
Hi everyone, I had this message but I don't know how to debug it. I have set apikey when I installed the plugin (with vim-plug) but it didn't work so I read somewhere that if I delete ~/.wakatime and run scripts/install_cli.py it might work. It didn't

After that I have the following message when I open vim:
![image](https://user-images.githubusercontent.com/9726654/178623352-b23108ff-4266-4ee5-961f-68b7e6e1c698.png)

Any idea of what to do? Am I missing any step? I 
- added plugin in .vimrc
- run :PlugInstall
- set my apikey from my account

I'm using osx (if that helps)

**Comment by alanhamlett** [comment]
Closing for now, but let me know if this error still happens and I'll re-open the issue.

**Comment by jeeeem** [comment]
Hijacking this thread.

![image](https://user-images.githubusercontent.com/51053482/203348783-aa80f067-7c84-4bc4-85d2-c468e2d29645.png)

I encountering this error and I tried running the script with `debug=true`.

I tried running the script before this picture and it installed successfully that why this is the result.

![image](https://user-images.githubusercontent.com/51053482/203349026-aa58764d-5f03-4b68-899e-2b2bf3fbc80c.png)

OS: Windows 11
Nvim: v0.9.0-dev-356+g6e8ed5aba
Shell: Powershell 7.3
